src/MessagePack.UnityClient/Assets/Scripts/MessagePack/MessagePackCompression.cs

```diff
@@ -25,5 +25,33 @@ public enum MessagePackCompression
         /// See also ThisLibraryExtensionTypeCodes.LZ4
         /// </devremarks>
         Lz4Block,
+
+        /// <summary>
+        /// Compresses an entire msgpack sequence as a multi contiguous block.
```

Comment:
The descriptions and names of the two enum values confuse me. "Contiguous" means it's "all together", but the unique value of this option is that it does *not* require the memory to be contiguous. Your description says it's "multi contiguous" which strikes me as a contradiction in terms. 
What if we called this `Lz4BlockArray`? or something like that that signifies there are multiple of them? The "Array" signifies the underlying msgpack format being an array.

Reply:
Well, it is used in the sense of concatenated Lz4BlockFormat, which is continuous as binary.
It ’s not memory contiguous.

I agree to use `Lz4BlockArray`.